netty-mqtt/src/test/java/org/thingsboard/mqtt/MqttClientTest.java

```diff
@@ -119,6 +119,36 @@ void testConnectToBroker() {
         assertThat(client.isConnected()).isTrue();
     }
 
+    @Test
+    void testDisconnectFromBroker() {
+        // GIVEN
+        var clientConfig = new MqttClientConfig();
+        clientConfig.setOwnerId("Test[ConnectToBroker]");
+        clientConfig.setClientId("connect");
```

Comment:
```suggestion
        clientConfig.setOwnerId("Test[Disconnect]");
        clientConfig.setClientId("disconnect");
```
Looks like copy-paste mistake